### PR TITLE
assorted fixes, rephrasing & improvement suggestions (instructor training checkout)

### PR DESCRIPTION
**Pull request by katrinleinweber**
Hello!

Within the [mentoring program](https://github.com/carpentries/mentoring/), and as preparation for an instructor training, I set myself the goal of working through this lesson and suggesting improvements (see commit messages).

Because several more commits may be added to this PR, I'm not sure how the review should be approached: intermediate approvals, or should I declare the PR ready once I feel it is? [GitLab uses `WIP` in MR titles](https://about.gitlab.com/2016/10/25/gitlab-workflow-an-overview/#wip-mr), which we can try here if you want.

Kind regards, and thanks for the mentoring program :-)

### CODE REVIEW

**Comment by iglpdc** [comment]
Thanks, @katrinleinweber !

**Comment by katrinleinweber** [comment]
Thanks! My instructor training is next week, so let's leave this open for further change suggestions.

**Comment by katrinleinweber** [comment]
OK, that's it for now. If the `b4aab37`, `e6da8be`& `d8f07e2` are fine as well, this is ready to be (rebased and/ or squashed and) merged IMHO :-)

**Comment by katrinleinweber** [comment]
Hello @iglpdc or @ErinBecker! This is fit to merge from my PoV. Do you agree/re-approve the last 3 commits? Many greetings, and happy holidays :-)

File: _episodes/04-changes.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ questions:
 - "How do I record notes about what changes I made and why?"
 objectives:
 - "Go through the modify-add-commit cycle for one or more files."
-- "Explain where information is stored at each stage of Git commit workflow."
+- "Explain where information is stored at each stage that cycle."
```
Comment (iglpdc) [inline _episodes/04-changes.md]: Missing "of"? "... each stage of that cycle."

**Comment by iglpdc** [comment]
Thank you very much, @katrinleinweber ! There is one minor comment above, but it's good for the rest. 

Another small thing: some of your changes affect files common to all lesson repos, such as `CONTRIBUTING.md` or `bin/lesson_initialize.py`. We pull those (once in a while) from [our lesson template](https://github.com/swcarpentry/styles). Would you like to add those patches there? It'll help all the other lessons too. 

You don't need to remove them from this PR: we'll handle it later when we merge from the template repo.

**Comment by iglpdc** [comment]
Thanks, @katrinleinweber !

**Comment by katrinleinweber** [comment]
You're welcome :-)

**Pull request merged**